### ISSUE TITLE
Path operator can't be used to construct absolute paths

### ISSUE THREAD

**Issue by wezm** [casey/just#1298]
It seemed fairly natural (to me) to try to construct an absolute path with the `/` operator, however this does not work.

E.g.

```just
remote_project_path :=  / "home" / user / project
```

yields

```
error: Expected backtick, identifier, '(', or string, but found '/'
   |
12 | remote_project_path := / "home" / user / project
   |                        ^
```

I tried wrapping it in parens in case that helped but it did not.

**Comment by casey** [comment]
Seems natural to me too! If anyone wants to grab this, it would make a great first issue.

**Comment by erikkrieg** [comment]
I took a stab at this change here: https://github.com/casey/just/pull/1320

**Comment by wezm** [comment]
Thanks so much for tackling this issue @erikkrieg 🎉

**Comment by casey** [comment]
Just released 1.5.0, which includes this change. Release binaries should be available shortly: https://github.com/casey/just/releases
